__tests__/hooks/useGroupSetting.test.js

```diff
@@ -233,30 +239,6 @@ describe('use groupSetting should run correctly', () => {
     expect(result.current.renderHeaderSubtitleText()).toEqual('Skip');
   });
 
-  it('submitData should run correctly', async () => {
-    const navigation = {
-      navigate: jest.fn(),
-      push: jest.fn(),
-      goBack: jest.fn()
-    };
-    const route = {
-      params: {
-        username: 'agita'
-      }
-    };
-    const spyUploadFile = jest
-      .spyOn(serviceFile, 'uploadFile')
-      .mockImplementation(() => ({data: {url: 'https://image.jpg'}}));
-    const {result} = renderHook(() => useGroupSetting({navigation, route}), {wrapper});
-    await result.current.setChangeImage(true);
-    await result.current.submitData(true, true);
-    expect(spyUploadFile).toHaveBeenCalled();
-    expect(result.current.isLoading).toBeFalsy();
-    await result.current.setChangeImage(false);
-    await result.current.submitData(true, false);
-    expect(navigation.goBack).toHaveBeenCalled();
-  });
-
   it('handleResLaunchGallery should run correctly', () => {
     const navigation = {
       navigate: jest.fn(),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test for 'submitData' has been removed. If this functionality is no longer present or has been moved elsewhere, this is fine. Otherwise, removing this test might decrease coverage.
```diff
-  it('submitData should run correctly', async () => {
-    const navigation = {
-      navigate: jest.fn(),
-      push: jest.fn(),
-      goBack: jest.fn()
-    };
-    const route = {
-      params: {
-        username: 'agita'
-      }
-    };
-    const spyUploadFile = jest
-      .spyOn(serviceFile, 'uploadFile')
-      .mockImplementation(() => ({data: {url: 'https://image.jpg'}}));
-    const {result} = renderHook(() => useGroupSetting({navigation, route}), {wrapper});
-    await result.current.setChangeImage(true);
-    await result.current.submitData(true, true);
-    expect(spyUploadFile).toHaveBeenCalled();
-    expect(result.current.isLoading).toBeFalsy();
-    await result.current.setChangeImage(false);
-    await result.current.submitData(true, false);
-    expect(navigation.goBack).toHaveBeenCalled();
-  });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->